GTMAppAuth/Sources/GTMAppAuthFetcherAuthorization+Keychain.m

```diff
@@ -23,7 +23,14 @@
 @implementation GTMAppAuthFetcherAuthorization (Keychain)
 
 + (GTMAppAuthFetcherAuthorization *)authorizationFromKeychainForName:(NSString *)keychainItemName {
-  NSData *passwordData = [GTMKeychain passwordDataFromKeychainForName:keychainItemName];
+  return [GTMAppAuthFetcherAuthorization authorizationFromKeychainForName:keychainItemName
+                                               withDataProtectionKeychain:NO];
+}
+
++ (GTMAppAuthFetcherAuthorization *)authorizationFromKeychainForName:(NSString *)keychainItemName
+                                          withDataProtectionKeychain:(BOOL)dataProtectionKeychain {
```

Comment:
Nit: I would expect a param name like `withDataProtectionKeychain` to take an argument of type `DataProtectionKeychain`. Since this is a `BOOL`, maybe use something like: `shouldUseDataProtectionKeychain`?

Reply:
Maybe `useDataProtectionKeychain`?

Reply:
Good point.  What if we simply shortened it to `dataProtectionKeychain:(BOOL)dataProtectionKeychain`?  I believe there's some precedent in the Cocoa APIs for that.